tests/core/test_trajectory.py

```diff
@@ -471,3 +471,7 @@ def test_xdatcar_write(self):
         written_traj = Trajectory.from_file("traj_test_XDATCAR")
         self._check_traj_equality(self.traj, written_traj)
         os.remove("traj_test_XDATCAR")
+
+    def test_read_file(self):
```

Comment:
best match the method name in the test name: i.e. `test_from_file`. Also, maybe check the traj length and composition of the 1st frame for extra safety.